src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -179,13 +185,8 @@ class AddressValidationModal extends React.Component {
       closeModal();
     };
 
-    const confirmedSuggestions = suggestedAddresses.filter(
```

Comment:
We were already doing this in the thunk, so I updated it to pass confirmedSuggestions to the component instead